tilty/emitters/webhook.py

```diff
@@ -43,18 +43,20 @@ def __init__(self, config: dict) -> None:
         self.delay_minutes: Union[int, None] = delay_minutes
         self.headers: dict = json.loads(config['headers'])
         self.template: Template = Template(config['payload_template'])
-        self.delay_until: Union[datetime.datetime, None] = None
+        self.delay_until: Dict[str, datetime.datetime] = dict()
 
-    def emit(self, tilt_data: dict) -> None:  # pylint:disable=inconsistent-return-statements  # noqa
+    def emit(self, tilt_data: dict) -> None:
         """ Initializer
 
         Args:
             tilt_data (dict): data returned from valid tilt device scan
         """
 
+        tilt_mac = str(tilt_data['mac'])
```

Comment:
if we dont hardcode 'mac' here but use a config item for 'color' or 'mac' we can let it work the lazy way, by color, but let it be per mac also

Reply:
After taking another look, I found a uuid, which should be great and wouldn't need additional config. I exposed that in scan_for_tilt_data (plus I fixed a couple issues with that function as well).

Reply:
yeah, uuid is about as useful as the color though since theyre 1:1 in constants.py